test/modules/corrado/bigint_inversion_error.chpl

```diff
@@ -0,0 +1,35 @@
+use BigInteger;
+
+// inverse exists
+var a = new bigint(0);
+var b = new bigint(5);
+var c = new bigint(2);
+
+// a = (b % c)^(-1)
+// a = (5 % 2)^(-1)
+// a = (1)^(-1)
+// a = 1
+var status = a.invert(b, c);
+
+writeln("Inverse Exists Case:");
+writeln("\tnon-zero status: ", status != 0);
+writeln("\tcorrect inverse: ", a == 1);
+
+// inverse does not exist (error case)
+var d = new bigint(3);
+var e = new bigint(100);
+var f = new bigint(10);
+
+writeln("Inverse Does Not Exist Case:");
+try {
+    // d = (e % f)^(-1)
+    // d = (100 % 10)^(-1)
+    // d = (0)^(-1)
+    // d = undefined
+    d.invert(e, f);    
+} catch e: InversionError {
+    writeln("\tcaught: ", e);
+} catch {
+    writeln("\tcaught unexpected error");
+}
+writeln("\toperand still defined: ", d == 3);
```

Comment:
Overall, I like this test!  This line is the only one I have issue with, though - the GMP function we're calling to implement this behavior, `mpz_invert`, itself says that the value put in `rop` will be undefined if there's no inverse.  But this line is checking that it's set to a specific value.  I don't think we should check that value at all in the case where the error is thrown.  I'd just delete it.